apps/authz/kinds/v0alpha1/rolespec.cue

```diff
@@ -0,0 +1,23 @@
+package v0alpha1
+
+RoleSpec: {
+	#Permission: {
+		// RBAC action (e.g: "dashbaords:read")
+		action: string
+		// RBAC scope (e.g: "dashboards:uid:dash1")
+		Scope: string
+	}
+	
+	name: string
+	displayName: string
+	uid: string
+	version: int
+	group: string
+	permissions: [...#Permission]
+
+	// TODO:
+	// delegatable?: bool
+	// created?
+	// updated?
```

Comment:
I'm not sure about those. I think created and updated is managed by the unified storage.
Hidden is probably going to be managed by label.
Delegatable 🤷